setup-env/action.yml

```diff
@@ -13,6 +13,14 @@ inputs:
   project-name:
     description: 'Project name for the tests'
     required: false
+  github-token:
```

Comment:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#about-the-github_token-secret:~:text=The%20token%20is%20also%20available%20in%20the%20github.token%20context.%20For%20more%20information%2C%20see%20%22Accessing%20contextual%20information%20about%20workflow%20runs.%22

github_token is available as part of github context as per above docs. Can we check once?

Reply:
yes, it is available, removed this